mmv1/templates/terraform/examples/dataform_repository_ssh.tf.erb

```diff
@@ -0,0 +1,42 @@
+resource "google_sourcerepo_repository" "git_repository" {
+  provider = google-beta
+  name = "<%= ctx[:vars]['git_repository_name'] %>"
+}
+
+resource "google_secret_manager_secret" "secret" {
+  provider = google-beta
+  secret_id = "secret"
+
+  replication {
+    auto {}
+  }
+}
+
+resource "google_secret_manager_secret_version" "secret_version" {
+  provider = google-beta
+  secret = google_secret_manager_secret.secret.id
+
+  secret_data = "<%= ctx[:vars]['data'] %>"
+}
+
+resource "google_dataform_repository" "<%= ctx[:primary_resource_id] %>_git_ssh_connection" {
```

Comment:
Nit: the `_git_ssh_connection` suffix can be removed now that this is in a dedicated example

Reply:
Done